eg/Classes/GUID.py

```diff
@@ -64,14 +76,25 @@ def NewId(self, target):
         return guid
 
     def AddId(self, target, guid):
-        guid = GUIDBase(target, guid)
-        self.guidObjects[guid.guid] = guid
-        return guid
+        if guid in self.guidObjects:
+            new = self.NewId(guid)
+            eg.PrintDebugMessage(
```

Comment:
Wow I put that in there right before I did the commit. That's funny. The one thing I didn't test. Lmao. That would figure. I'll make the change and commit again